tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -13414,7 +13417,7 @@ bool Sema::DiagnoseHLSLDecl(Declarator &D, DeclContext *DC, Expr *BitWidth,
 
   // Disallow bitfields where not enabled explicitly or by HV
   if (BitWidth) {
-    if (!getLangOpts().EnableBitfields) {
+    if (getLangOpts().HLSLVersion < hlsl::LangStd::v2021) {
```

Comment:
For @llvm-beanz benefit, this is where I fixed the merge because the previously joint conditional was split.